bindings/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "packageName": "determined.swagger.client"
```

Comment:
Let's put this at `determined._swagger.client`, to make it excessively clear that it is not a public api